doc/api/dns.md

```diff
@@ -7,31 +7,13 @@ MeasurementKit (libmeasurement_kit, -lmeasurement_kit).
 # SYNOPSIS
 ```C++
 #include <measurement_kit/dns.hpp>
-
-// Constructs resolver with default settings
-mk::dns::Resolver resolver;
-
-// Constructs resolver with specific settings
-mk::dns::Resolver reso({
-    {"nameserver", "8.8.8.8:53"},  // Set the name server IP
-    {"attempts", "1"},             // How many attempts before erroring out
-    {"timeout", "3.1415"},         // How many seconds before timeout
-    {"randomize_case", "1"},       // Whether to randomize request case
-});
-
-// Issue an async DNS query
-reso.query(
-        "IN",                                             // Domain of the query
-        "AAAA",                                           // Type of query
-        "nexa.polito.it",                                 // Name to resolve
-        [](mk::Error error, mk::dns::Response response) { // Callback
-            if (error) throw error;
-            // handle successful response
-        });
```

Comment:
I'd not remove this part of the synopsis. See how HTTP documentation now looks like.